src/tmp_structs.h

```diff
@@ -0,0 +1,10 @@
+#ifndef TMP_STRUCTS_H
+#define TMP_STRUCTS_H
+
+#include "../include/raylib.h"
+
+typedef struct StateManager{
```

Comment:
I think this is redundant. Could we just have one variable called `state`, which can store an integer? Maybe we can further discuss this in the next meeting.

Reply:
Maybe try this!

```c
typedef unsigned short State;
// ...
State state = INITIAL_STATE;
```

Reply:
I have added a header file called `start.h` which holds the different values for the states